cds-security-mt.json

```diff
@@ -2,29 +2,39 @@
 	"xsappname": "bookshop-java-public",
 	"tenant-mode": "shared",
 	"scopes": [
-	  {
-		"name": "$XSAPPNAME.admin",
-		"description": "admin"
-	  },
-	  {
-		"name": "$XSAPPNAME.mtcallback",
-		"description": "Multi Tenancy Callback Access",
-		"grant-as-authority-to-apps": ["$XSAPPNAME(application,sap-provisioning,tenant-onboarding)"]
-	  },
-	  {
-		  "name": "$XSAPPNAME.mtdeployment",
-		  "description": "Scope to trigger a re-deployment of the database artifacts"
-	  }
+		{
+			"name": "$XSAPPNAME.admin",
+			"description": "admin"
+		},
+		{
+			"name": "$XSAPPNAME.mtcallback",
+			"description": "Multi Tenancy Callback Access",
+			"grant-as-authority-to-apps": [
+				"$XSAPPNAME(application,sap-provisioning,tenant-onboarding)"
+			]
+		},
+		{
+			"name": "$XSAPPNAME.mtdeployment",
+			"description": "Scope to trigger a re-deployment of the database artifacts"
+		},
+		{
+			"name": "$XSAPPNAME.ExtendCDS",
+			"description": "Extend tenant models"
```

Comment:
Maybe make this more clear that this grants extension rights for a specific tenant. Maybe something like `Extend your own tenant's models`